paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -66,15 +66,19 @@ abstract class PrepareResourcesTask : DefaultTask() {
   private val projectDirectory = project.layout.projectDirectory
 
   @TaskAction
+  @OptIn(ExperimentalStdlibApi::class)
```

Comment:
@JakeWharton, correct if I'm wrong, but I think you had said in the past (maybe in a Timber PR?) that using experimental apis, like buildList etc. in libraries, isn't ideal.  Is that guidance still accurate?

Reply:
There where already some instances of `ExperimentalStdlibApi`, one example being https://github.com/cashapp/paparazzi/blob/9fcffd16467697d85cb4518a188cc7fbfab93092/paparazzi/src/main/java/app/cash/paparazzi/internal/parsers/ResourceParser.kt#L124-L126
That's why I included it here. 

Apart from that, are there any changes required?

Reply:
fair enough.  i forgot we used that elsewhere.  we can migrate later then, if an issue!